src/gleam/utf_codepoint.gleam

```diff
@@ -0,0 +1,21 @@
+import gleam/result
+
+/// A UtfCodepoint is the integer representation of a valid UTF codepoint
+pub type UtfCodepoint = UtfCodepoint
```

Comment:
Has this been formatted?

Reply:
Apparently not, sorry. I ran `gleam format` but it didn't actually do anything. Looks like I needed to explicitly pass in the directory. I would expect it to work on the current directory by default, no?

Reply:
Yeah good idea, I'll make an issue